data/simple-icons.json

```
@@ -11382,6 +11382,20 @@
 		"source": "https://en.facebookbrand.com/oculus/assets/oculus?audience=oculus-landing",
 		"guidelines": "https://en.facebookbrand.com/oculus/"
 	},
+	{
+		"title": "Odido",
+		"hex": "000000",
+		"source": "https://www.odido.nl/Assets/Lfwd/Static/odido-logo-black.svg",
```

For better future proofing, we use the URL of the page the source file was found but, in this case, the logo page of the brand guidelines works better.
```suggestion
		"source": "https://merk.odido.nl/en/logo",
```